docs/DeveloperGuide.md

```diff
@@ -1,14 +1,4 @@
-# AddressBook Level 4 - Developer Guide
-
-By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbsp;&nbsp; Licence: `MIT`
-
----
-
-1. [Setting Up](#setting-up)
-2. [Design](#design)
```

Comment:
Remember to update the `Design` when you make changes to the code base.
You don't want to push the documentations thing in the end right?